doc/openapi-user-guide.md

```diff
@@ -92,6 +93,77 @@ spec:
 
 [OpenAPI CRD Reference](openapi-reference.md) for more info.
 
+### OpenID Connect and OAuth2 example
+
+3scale requires some additional information that are not included 
+in the OpenAPI spec, therefore those data need to be provided in the OpenAPI CR. Specifically:
+- OpenID Connect Issuer Type, which will default to rest but can be overridden from the OpenAPI CR
+- OpenID Connect Issuer Endpoint Referense (Secret). 3scale requires that the issuer URL must include a client secret.
+- Flows object. When the sec scheme is oauth2, the flows are provided by the OpenAPI doc. However, for openIdConnect security scheme, the OpenAPI doc does not provide the flows. In that case, the OpenAPI CR can provide those.
+
+#### OIDC Issuer Secret
+
+```yaml
+kind: Secret
+apiVersion: v1
+metadata:
+  name: my-secret
+  namespace: 3scale-test
+data:
+  issuerEndpoint: https://3scale-zync:some-secret@keycloak-rhsso-test.apps.xxxxx.xxxx.s1.devshift.org/auth/realms/petstore
+type: Opaque
+```
+To create secret:
+```shell
+oc apply -f my-secret.yaml 
+```
+
+**NOTE** 
+- The secret value (*some-secret* in this example) will be taken from the Issuer Client secret. If you are using RHSSO - the secret could be found at `Realm/Clients/ClientID/Credentials/Secret`.
```

Comment:
think this should be *my-secret* in this example

Reply:
updated; it was not clear, hope better now. Thank you for comment!